mrp_bom_product_details/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
```

Comment:
Change this line by

`:target: https://www.gnu.org/licenses/agpl`

as is indicated in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst